### ISSUE TITLE
A low number of iterations in fastmode decreases accuracy

### ISSUE THREAD

**Issue by gabraham** [DReichLab/EIG#5]
Hi, you may wish to increase the default number of iterations in fastpca (`fastiter`), for some datasets the default is too low, resulting in inaccurate estimation of eigenvectors (I checked this by comparing the non-fastmode results with the fastmode, and verified with several other SVD implementations). 

For example with the default iterations:
```
> diag(cor(P1, P2))
        V2         V3         V4         V5         V6         V7         V8
 0.9985658  0.9909673  0.9819172 -0.9131868  0.9447627  0.9555776 -0.8435918
        V9        V10        V11
-0.7538453 -0.6555316  0.8961565
```

With `fastiter: 100`
```
> diag(cor(P1, P2))
 V2  V3  V4  V5  V6  V7  V8  V9 V10 V11
  1   1   1  -1  -1  -1  -1   1   1   1
```
where `P1` and `P2` are the top ten eigenvectors from non-fastmode and fastmode, respectively. The dataset consists of 5300 individuals with 3700 LD-thinned SNPs.

I'm using v6.1.1.

**Comment by bumblenick** [comment]
Thanks;  Of course if eigenvalues are very close, you can expect some
ambiguity about what is returned
(and in practice if the space spanned is OK, users probably don't care).

And the default is just that.   Not universally good.  But there may be
some way to monitor convergence and
we should think about that.   By the way the # of SNPs is very
(unreasonably??) small for so many individuals.

Nick

**Comment by gabraham** [comment]
Yes I realise that there is no "universally good" number of iterations (my "competing" method, https://github.com/gabraham/flashpca, checks for one form of convergence).

I only used 3700 SNPs in this example for the sake of speed, I see the same behaviour for larger datasets with more SNPs.
